src/main/java/com/robertx22/age_of_exile/gui/screens/skill_tree/SkillTreeScreen.java

```diff
@@ -240,47 +269,6 @@ public void addButtonPublic(AbstractWidget b) {
         this.addRenderableWidget(b);
     }
 
-    private void addConnections() {
```

Comment:
where did this go?

Reply:
completely replaced by the connection cache.